tests/integration/test_agent.py

```diff
@@ -21,32 +21,19 @@
 workspace_base = os.getenv('WORKSPACE_BASE')
 workspace_mount_path = os.getenv('WORKSPACE_MOUNT_PATH')
 workspace_mount_path_in_sandbox = os.getenv('WORKSPACE_MOUNT_PATH_IN_SANDBOX')
+max_iterations = 15
+max_budget_per_task = 15
 
 print('\nPaths used:')
 print(f'workspace_base: {workspace_base}')
 print(f'workspace_mount_path: {workspace_mount_path}')
 print(f'workspace_mount_path_in_sandbox: {workspace_mount_path_in_sandbox}')
 
 
-def get_number_of_prompts(test_name: str):
-    mock_dir = os.path.join(
-        os.environ.get('SCRIPT_DIR'), 'mock', os.environ.get('DEFAULT_AGENT'), test_name
-    )
-    prompt_files = [file for file in os.listdir(mock_dir) if file.startswith('prompt_')]
-    return len(prompt_files)
-
-
-def validate_final_state(final_state: State | None, test_name: str):
+def validate_final_state(final_state: State | None):
     assert final_state is not None
     assert final_state.agent_state == AgentState.STOPPED
     assert final_state.last_error is None
-    # number of LLM conversations should be the same as number of prompt/response
-    # log files under mock/[agent]/[test_name] folder. If not, it means there are
-    # redundant prompt/response log files checked into the repository.
-    num_of_conversations = get_number_of_prompts(test_name)
-    assert num_of_conversations > 0
-    # we mock the cost of every conversation to be 1 USD
-    assert final_state.metrics.accumulated_cost == num_of_conversations
```

Comment:
This removal seems to due to problematic merge conflict resolution

Reply:
Woops, sorry, thanks for the catch!